code/src/components/TodoItem.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { todos } from "reducers/todos";
+
+
+export const TodoItem = ({itemIndex}) => {
+  const item = useSelector((store) => store.todos.list.items[itemIndex]);
```

Comment:
you could also just pass in the whole item as a prop to this component instead of fetching it again from the global store...